### DISCUSSION TITLE
QFieldCloud availability for Android

### DISCUSSION THREAD

**Discussion by saguaroland1939** [opengisch/QField#2489] [Q&A]
Hello from Phoenix, Arizona!

So I’m trying to enlist a collaborator for my QFieldCloud project, but when she opens the QField app on her Android phone the QFieldCloud button is grayed out and says the functionality is coming soon. Can anyone confirm that QFieldCloud is not available yet for Android? If that is the case, do you know when it will become available?

Many thanks,
Veronica

**Comment by suricactus** [comment]
Good morning, USA! 

@saguaroland1939  you need to install either BETA or early access (nightly/development) version of QField.

You can install nightly from here: https://play.google.com/store/apps/details?id=ch.opengis.qfield_dev&hl=en&gl=US

Best,
Ivan

**Reply by saguaroland1939** [reply]
Thanks Ivan! My collaborator installed this early access version and that did the trick.

-Veronica

**Reply by suricactus** [reply]
Glad to get your issue solved, :cactus: -land1939 !
